spec/models/manageiq/providers/ansible_tower/automation_manager/refresher_spec.rb

```diff
@@ -85,20 +85,25 @@ def assert_credentials
       :name   => "Demo Credential",
       :userid => "admin",
     )
+    expect(machine_credential.options.keys).to match_array(machine_credential.class::EXTRA_ATTRIBUTES.keys)
```

Comment:
can you also test for the values here and in the other 2 tests?
Or are they empty?
If empty, maybe change to creds that are not empty?